model/knn/stats/structures.smithy

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@mixin
+structure KNNStats_QueryParams {
+    @httpQuery("timeout")
+    timeout: Timeout,
```

Comment:
I see `pretty` querystring param in the examples as well.

Reply:
Theo, pretty is a [global parameter](https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate_api.py#L61).

Those parameters can be used with all APIs. So, we won't write them in spec. Please confirm?